src/info/title.rs

```diff
@@ -81,3 +81,72 @@ impl std::fmt::Display for Title {
         }
     }
 }
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use git_repository::{open, Repository, ThreadSafeRepository};
+    use git_testtools;
+    use owo_colors::AnsiColors;
+
+    type Result<T = ()> = std::result::Result<T, Box<dyn std::error::Error>>;
+
+    fn basic_repo() -> Result<Repository> {
+        let name = "basic_repo.sh".to_string();
+        let repo_path = git_testtools::scripted_fixture_repo_read_only(name)?;
+        let safe_repo = ThreadSafeRepository::open_opts(repo_path, open::Options::isolated())?;
+        Ok(safe_repo.to_thread_local())
+    }
+
+    #[test]
+    fn test_get_git_username() -> Result<()> {
```

Comment:
```suggestion
    fn test_get_git_username() -> Result {
```
Since you set `T` to default to `()`, you should be able to remove these.

Otherwise, if you feel like `T` should *always* be specified, I'd recommend removing the type's default above.